server/src/config.ts

```diff
@@ -13,7 +11,7 @@ export class Config {
   public static TOKEN_EXPIRATION: string = "1d";
   public static TOKEN_ISSUER: string = "socobo";
   public static TOKEN_HEADER: string = "x-access-token";
-  public static DEFAULT_USER_IMAGE: string = "http://placehold.it/350x150";
+  public static DEFAULT_USER_IMAGE_ID: number = 1;
```

Comment:
I think we should not rely on the database id since it is much harder to change the url of the image. I liked the url in the config file and would not mix the default image with user images. What do you think is the benefit of storing the url in the db?